File: src/Data/Char.js
/* global exports */
"use strict";

// module Data.Char

exports.toString = function (c) {
  return c;
};

exports.toCharCode = function (c) {
  return c.charCodeAt(0);
};

exports.fromCharCode = function (c) {
  return String.fromCharCode(c);
};

exports.toLowerChar = function (c) {
  return c.toLowerCase();
};

exports.toUpperChar = function (c) {
  return c.toUpperCase();
};

File: src/Data/Char.erl
-module(data_char@foreign).
-export([toCharCode/1, fromCharCode/1, toLower/1, toUpper/1]).

toCharCode(C) -> C.
fromCharCode(N) -> N.
toLower(C) -> string:to_lower(C).
toUpper(C) -> string:to_upper(C).
